installer/bundle/bundle_skel.sh

```diff
@@ -941,13 +941,31 @@ if [ "$installMode" = "R" -o "$installMode" = "P" ]; then
         fi
 
         pkg_rm omsconfig
-        pkg_rm omsagent
 
-        # If MDSD is installed and we're just removing (not purging), leave SCX
-        MDSD_INSTALLED=1
-        check_if_program_exists_on_system azsec-mdsd
-        if [ $? -eq 0 -o -d /var/lib/waagent/Microsoft.OSTCExtensions.LinuxDiagnostic-*/mdsd ]; then
-            MDSD_INSTALLED=0
+        # If MDSD/LAD is installed and we're just removing (not purging), leave OMS, SCX and OMI
+        check_if_pkg_is_installed azsec-mdsd
+        azsec_mdsd_installed=$?
+        check_if_pkg_is_installed lad-mdsd
+        lad_mdsd_installed=$?
+
+        # if at least one of mdsd product is installed
+        MDSD_INSTALLED=$(( $azsec_mdsd_installed && $lad_mdsd_installed ))
```

Comment:
I have seen the case where only one of them is installed and it could still break OMS uninstall

Reply:
comment says we are checking for "At least one," but we are doing a &&

Reply:
because the value "0" means installed, so I have to use XOR or AND.